src/Validation.elm

```diff
@@ -165,60 +159,79 @@ field value =
 
 
 {-| Default setting of Field with `Valid a` validity.
+For `Field String String` use an identity function.
 
     import Validation exposing (Field, field)
 
     intValue : Field String Int
     intValue =
-        preValidatedField "50"
+        preValidatedField String.fromInt 50
 
 
     -- Field "50" (Valid 50)
 
 -}
-preValidatedField : val -> Field String val
-preValidatedField value =
-    Field (toString value) (Valid value)
+preValidatedField : (val -> String) -> val -> Field String val
+preValidatedField fn value =
+    Field (fn value) (Valid value)
 
 
 {-| Run validation on Field with `Event`.
 -}
 validate : Event raw -> Validator raw a -> Field raw a -> Field raw a
-validate event validate (Field value validity) =
+validate event letValidate (Field value validita) =
```

Comment:
Co tady vadilo, ze jsou prejmenovane parametry?

Reply:
jj, kolidovalo to s fci validate